src/components/PassphraseExample.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import cx from 'classnames'
+import styles from 'styles/passphrase'
+
+const alphaRegexp = /^[a-z]$/i
+const numberRegexp = /^[0-9]$/
+
+const isAlphaCharacter = char => alphaRegexp.test(char)
+const isNumberCharacter = char => numberRegexp.test(char)
+
+const tokenTypes = {
+  alpha: 'alpha',
+  number: 'number',
+  special: 'special'
+}
+
+const getTokenType = char => {
+  if (isAlphaCharacter(char)) {
+    return tokenTypes.alpha
+  }
+
+  if (isNumberCharacter(char)) {
+    return tokenTypes.number
+  }
+
+  return tokenTypes.special
+}
```

Comment:
Why can we have several types of token?

Reply:
Because we want to color alpha, numeric and others (considered as special) characters in the password example differently:

![image](https://user-images.githubusercontent.com/1606068/70727973-e2c52d80-1d00-11ea-93f1-d3e1342fc091.png)

So I cut the string and give a type to each char

Reply:
Got it! Thanks. It makes more sense now :)